docker-compose.yml

```diff
@@ -14,7 +14,7 @@ services:
       test: [ "CMD", "curl", "-f", "http://localhost:8080/health/ping" ]
     environment:
       - SERVER_PORT=8080
-      - OAUTH_API_BASE_URL=http://localhost:9090/auth
+      - OAUTH_API_BASE_URL=http://localhost:8080/auth
```

Comment:
I think this can stay as `9090` as that is the `hmpps-auth` container's published port

Reply:
good spot, think I was a bit hasty